src/Common/SurfaceFluxes/SurfaceFluxes.jl

```diff
@@ -271,6 +272,40 @@ function compute_physical_scale(
     return _π_group⁻¹ * _von_karman_const / Σterms * (x_in - x_s)
 end
 
+"""
+    recover_profile(z, x_star, x_s, z_0, L_MO, transport, ::DGScheme, uf)
```

Comment:
Can we improve the name here? I'm not sure what profile this refers to

Reply:
I am referring to u(z), θ(z), ... inside the computational domain, so the vertical profiles of the variables. Would `recover_vert_profiles` work better?

Reply:
Ok, maybe just clarify what's being returned in the doc string